pyhive/presto.py

```diff
@@ -80,7 +82,10 @@ class Cursor(common.DBAPICursor):
 
     def __init__(self, host, port='8080', username=None, catalog='hive',
                  schema='default', poll_interval=1, source='pyhive', session_props=None,
-                 protocol='http', password=None, requests_session=None, requests_kwargs=None):
+                 protocol='http', password=None, KerberosRemoteServiceName=None,
```

Comment:
This would break compatibility if people were using it without named argument. All the kerberos parameters can go at the end.

Reply:
Incorporated these changes.